src/components/snapshots/takeSnapshot.ts

```diff
@@ -25,18 +25,22 @@ import {
   PREFERENCES_FILE,
   PG_DFF_JS_LOCATION,
   DATABASE_FOLDER,
-} from './constants'
+} from '../../constants'
 import { getBaseFiles, getDirectoryFromPath } from './useSnapshot'
 import config from '../../config'
 import { DateTime } from 'luxon'
 const asyncRimRaf = promisify(rimraf)
+import { createDefaultDataFolders } from '../files/createDefaultFolders'
 
 const takeSnapshot: SnapshotOperation = async ({
   snapshotName = DEFAULT_SNAPSHOT_NAME,
   optionsName = DEFAULT_OPTIONS_NAME,
   options: inOptions,
   extraOptions = {},
 }) => {
+  // Ensure relevant folders exist
+  createDefaultDataFolders()
+
```

Comment:
To prevent errors in case the data folders don't exist yet. (This shouldn't happen, it's just a failsafe)